src/main/java/hudson/plugin/versioncolumn/JVMVersionMonitor.java

```diff
@@ -46,19 +52,31 @@ public class JVMVersionMonitor extends NodeMonitor {
 
     private JVMVersionComparator.ComparisonMode comparisonMode =
             JVMVersionComparator.ComparisonMode.RUNTIME_GREATER_OR_EQUAL_MASTER_BYTECODE;
-    private boolean disconnect = true;
+    private transient Boolean disconnect;
 
     @DataBoundConstructor
-    public JVMVersionMonitor(JVMVersionComparator.ComparisonMode comparisonMode, boolean disconnect) {
+    public JVMVersionMonitor(JVMVersionComparator.ComparisonMode comparisonMode) {
         this.comparisonMode = comparisonMode;
-        this.disconnect = disconnect;
     }
 
     public JVMVersionMonitor() {}
 
     @SuppressWarnings("unused") // jelly
     public boolean isDisconnect() {
-        return disconnect;
+        return !isIgnored();
+    }
+
+    // should be restricted/deprecated but that breaks casc
+    @DataBoundSetter
+    public void setDisconnect(boolean disconnect) {
```

Comment:
That method is required for backward compatibility with CasC. Ideally it should be deprecated (likewise isDisconnected should also be deprecated). But when doing so CasC will fail unless one adds
```
configuration-as-code:
  version: 1
  deprecated: warn
```
to the CasC config.
Similarly using `@Restricted(DoNotUse.class)` makes CasC fail